contentcuration/contentcuration/models.py

```diff
@@ -1243,6 +1257,10 @@ class ContentNode(MPTTModel, models.Model):
     # this duration should be in seconds.
     suggested_duration = models.IntegerField(blank=True, null=True, help_text="Suggested duration for the content node (in seconds)")
 
+    # A field to store the ts_vector form of (title + ' ' + description).
+    # This significantly increases the search performance.
+    title_description_search_vector = SearchVectorField(blank=True, null=True)
```

Comment:
Just to check my understanding - we are adding a column that basically acts like an arbitrary search index, right?

Reply:
Absolutely sir!